requirements/base.in

```diff
@@ -1,19 +1,20 @@
 -c "constraints.in"
+backoff
+beautifulsoup4
 chardet
+dataclasses-json
+emoji
 filetype
-python-magic
+langdetect
 lxml
 nltk
-tabulate
-requests
-beautifulsoup4
-emoji
-dataclasses-json
-python-iso639
-langdetect
 numpy
+python-iso639
+python-magic
 rapidfuzz
-backoff
+requests
+tabulate
 typing-extensions
-unstructured-client
+# NOTE(jennings): pinned due to later versions not supporting api_key_auth in UnstructuredClient
+unstructured-client>=0.9.0
```

Comment:
why not a higher version, like 0.15.1?

Reply:
I thought the minimum was preferred, but am happy to bump it up if that is preferred.